src/qmflows/backports.py

```diff
@@ -30,11 +30,17 @@ class _NullContextBackup(AbstractContextManager):
     Used as a stand-in for a normal context manager, when a particular
     block of code is only sometimes used with a normal context manager:
 
+    .. testsetup:: python
+
+        >>> cm = ...
```

Comment:
what is the meaning of thoese 3 dots there?

Reply:
Ah good catch, that line of code is actually completely redundant.

Reply:
Removed in https://github.com/SCM-NV/qmflows/pull/207/commits/054cb6e16a5199f7afcb986f21856a535ffdc93c.